### ISSUE TITLE
[custom-property-pattern] Script error with empty "var()"

### ISSUE THREAD

**Issue by danielrentz** [stylelint/stylelint#5987]
### What steps are needed to reproduce the bug?

Have a CSS rule with empty `var()`.

I encountered this bug when typing rules in VS Code where an error box pops up everytime when typing "var()" and trying to paste a variable name from clipboard afterwards.

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "custom-property-pattern": ".*"
  }
}
```

### How did you run Stylelint?

CLI, VS Code plugin

### Which version of Stylelint are you using?

14.6.0

### What did you expect to happen?

No script errors from linters. Rule needs to be silent, or even better should report a real lint error for the invalid empty "var()".

### What actually happened?

test.css
```css
#id {
    color: var();
}
```
results in
```
> yarn stylelint *.css
yarn run v1.22.10
$ ./node_modules/.bin/stylelint *.css
TypeError: Cannot destructure property 'value' of 'nodes[0]' as it is undefined.
    at ./test.css:2:5
    at ./node_modules/stylelint/lib/rules/custom-property-pattern/index.js:61:20
    at walk (./node_modules/postcss-value-parser/lib/walk.js:7:16)
    at ValueParser.walk (./node_modules/postcss-value-parser/lib/index.js:18:3)
    at ./node_modules/stylelint/lib/rules/custom-property-pattern/index.js:54:16
    at ./node_modules/postcss/lib/container.js:72:18
    at ./node_modules/postcss/lib/container.js:55:18
    at Rule.each (./node_modules/postcss/lib/container.js:41:16)
    at Rule.walk (./node_modules/postcss/lib/container.js:52:17)
    at ./node_modules/postcss/lib/container.js:60:24
    at Root.each (./node_modules/postcss/lib/container.js:41:16)
error Command failed with exit code 1.
```

### Does the bug relate to non-standard syntax?

no

**Comment by ybiquitous** [comment]
@danielrentz Thanks for the report. It seems a duplicate of #5972, and v14.6.1 (#5986) will resolve it.

**Comment by jeddy3** [comment]
@danielrentz Thanks for using the template.

> It seems a duplicate of https://github.com/stylelint/stylelint/issues/5972, and v14.6.1 (https://github.com/stylelint/stylelint/issues/5986) will resolve it.

That's corrected. It's fixed in 14.6.1 that was just releaesd.
